src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -122,9 +134,9 @@ echo "{}" > esy.json
 
 **Your project is ready!**
 
-### FA2 contract
+### Step 2: FA2 contract
 
-We will rely on the Ligo FA library. To understand in detail how assets work on Tezos, please read below notes:
+Next, we will build the FA2 contract, which relies on the Ligo FA library. To understand in detail how assets work on Tezos, please read the notes below.
```

Comment:
"Next, we will build the FA2 contract, which relies on the Ligo FA library." This doesn't tell me what I'm actually doing in terms of the tutorial. Why am I building a contract?